types/driver.go

```diff
@@ -32,7 +32,7 @@ func (driver *Driver) Start() {
 	totalQuestions := round.TotalQuestions
 
 	for currentQuestion := 0; currentQuestion < totalQuestions; currentQuestion++ {
-
+		clearScreen()
```

Comment:
@learning-dev Is this method call still necessary? I guess printing ```fmt.Println("\033[H\033[2J")``` should do that. And if the fmt statement is working I guess we should do it before question is printed.

Reply:
No, it's not. I got into some merge issue. The old change and new change are present. I am trying to resolve it.

Reply:
can you check now?

Reply:
Looks good! Please move up the fmt.Println to before the question is being generated (to line 35) as it will refresh the screen once question is complete. After this I guess, the PR is ready to be merged. :smile:

Reply:
No, I have intentionally Placed it at that line. If not then screen will be cleared before printing the score.  By this you can see the score as well the new question in the screen. If not, you will see the new question but miss out the score every time.

Reply:
Alright, I missed out on that. Thanks!